scripts/r.in.srtm/r.in.srtm.py

```diff
@@ -270,7 +264,7 @@ def main():
     outf.close()
 
     try:
-        grass.run_command('r.in.gdal', input=bilfile, out=tileout)
+        grass.run_command('r.import', input=bilfile, out=tileout, resample='bilinear')
```

Comment:
```suggestion
        grass.run_command('r.import', input=bilfile, out=tileout, resample='bilinear_f')
```

Reply:
I understand the desire to make things easier for users, but at the same time, choice of resampling methods used during reprojection can have significant impacts on some applications. I am always weary, therefore, of such "black box" actions in modules. My favorite solution is to just keep it as is. If someone needs to reproject, they can do so and should then think about the relevant solution. My next favorite solution would be to make resampling algorithm a parameter or r.in.srtm. Finally, if this is really against other people's desires, then please at least make this into a big fat, very visible information in the r.in.srtm man page, informing users of the choice made for algorithm and that they can use the classic path (import into latlong, then reproject) to get more choice.

I know that I'm on the losing end of this debate in our community, but in my eyes the increased use of r.import has really as many disadvantages as advantages and I've already seen several situations where has created issues because people weren't even really aware that they were reprojecting, and definitely not aware of the consequences this can have, especially on raster data. It used to be GRASS GIS strength that it forces users to think more consciously about their data. We are slowly moving away from that, sacrificing rigor for "ease of use" (although this sometimes is immediate ease vs long-term ease). I hope that the gain in terms of uptake is worth the risk...

Reply:
I understand and agree, but I am afraid at this point people expect simple workflows. So we need to have reasonable defaults (ideally with easy access to change them). What r.import could do, is to decide best resampling based on data type (CELL - nearest neighbor, FCELL/DCELL - bilinear). This would at least help to prevent the case when continuous data such as elevation get resampled with NN. Or at least print a warning.

Reply:
Please keep in mind that this PR is about r.in.srtm.
I agree to the suggestion to make `resample` a user option, and then set it to `bilinear_f` as the default (while providing the possible `resample` answers of `r.import` as choices).

Reply:
> I understand and agree, but I am afraid at this point people expect simple workflows.

But workflows become even more complicated, see PROJ 6 with its several different coordinate operations depending on the area of interest (computational region in GRASS). Users have to decide, not software.

> So we need to have reasonable defaults (ideally with easy access to change them). What r.import could do, is to decide best resampling based on data type (CELL - nearest neighbor, FCELL/DCELL - bilinear). This would at least help to prevent the case when continuous data such as elevation get resampled with NN. Or at least print a warning.

E.g. SRTM is of type CELL, thus you are contradicting yourself: "use nearest neighbor for CELL but prevent the case when continuous data such as elevation get resampled with NN."

The datatype does not matter, what matters is the meaning of the data: metric, ordinal, or categorical. Only the user knows and thus has to decide on the best resampling method.

Regarding r.in.srtm, bilinear is a good choice if source and target resolution are similar. If the target resolution is much smaller, bicubic and lanczos will give better results. If the target resolution is much larger, NN should be used with a target resolution similar to the source resolution, followed by r.resamp.stats to get to the final target resolution.

Reply:
Type can be a proxy to meaning of data, maybe not a great one, but the only we have. I am trying to suggest reasonable defaults that do minimum harm to the data. Many users won't know how to pick the method and just go with default and I've seen problems coming from that. I agree there are situations when reasonable defaults might not be available but I don't think this is the case. So the question is would my suggestion introduce more problems comparing to current state of r.import considering usual (and less usual) use cases?
From what I understood from ML discussion on PROJ6, they offer reasonable defaults too. (Sorry for hijacking this PR...)